src/Services/Catalog/Catalog.Infrastructure/Services/ImageService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Catalog.Application.Services;
+using Catalog.Domain.Abstractions;
+using Catalog.Domain.Entities;
+using Catalog.Infrastructure.Options;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+
+namespace Catalog.Infrastructure.Services;
+
+public class ImageService : IImageService
```

Comment:
services should be located in application layer

Reply:
I thought that all classes, that implement services and deal with external systems and frameworks, such as file system or db, should be placed in infrastructure layer. If I'm wrong, please tell me your thoughts about this